File: URI/Python/1046.py
inicial, final = input().split()

inicial = int(inicial)
final = int(final)

if inicial < final:
	print("O JOGO DUROU {} HORA(S)".format(final-inicial))
elif inicial > final:
	print("O JOGO DUROU {} HORA(S)".format(24 - inicial + final))
elif inicial == final:
	print("O JOGO DUROU 24 HORA(S)")

File: URI/C++/1046.cpp
#include<iostream>
using namespace std;

int main(){
	int inicio, fim, total = 0;

	cin >> inicio >> fim;

	if(inicio == fim)
		total = 24;
	else if(inicio < fim)
		total = fim - inicio;
	else
		total = (24 - inicio) + fim;

	cout << "O JOGO DUROU "<< total <<" HORA(S)\n";


	return 0;
}